app/jobs/prepare_move_notifications_job.rb

```diff
@@ -56,6 +56,6 @@ def event_type(action_name)
       'update' => 'update_move',
       'update_status' => 'update_move_status',
       'destroy' => 'destroy_move',
-    }[action_name]
+    }[action_name] || action_name
```

Comment:
What's the reason you chose not to add `confirm_person_escort_record` to the as a key, I guess its not related to move?

Reply:
I decided it was fairly pointless adding a mapping for `confirm_person_escort_record` here, and instead allowed the event type to be passed in directly by adding this fallback. The existing mappings are for move related actions; the PER completion is piggy backing onto this functionality but doesn't fit with the same model of actions really.